### ISSUE TITLE
Regex Performance Improvement

### ISSUE THREAD

**Issue by forewarned** [nturley3/zeek-http-suspect-data-exposure#1]
Because of the amount of regex, this script could consume a lot of resources on Zeek. @JustinAzoff suggested combining all regex in the various files into a single expression, then if there is a hit to check the individual expressions.

**Comment by JustinAzoff** [comment]
Yes, there's also another optimization that can be done and avoid using http_entity_data.

You can see an example of that here https://github.com/corelight/CVE-2021-41773/blob/master/scripts/http_body_analysis.zeek

using a file analyzer allows you to remove the file analyzer after enough data has been seen, so you can avoid running a regex on an entire 20GB download.
